Corona-Warn-App/src/main/java/de/rki/coronawarnapp/dccticketing/core/service/processor/DccTicketingRequestServiceHelpers.kt

```diff
@@ -4,8 +4,27 @@ import de.rki.coronawarnapp.dccticketing.core.common.DccTicketingErrorCode
 import de.rki.coronawarnapp.dccticketing.core.common.DccTicketingException
 import de.rki.coronawarnapp.dccticketing.core.transaction.DccJWK
 import de.rki.coronawarnapp.dccticketing.core.transaction.DccTicketingServiceIdentityDocument
+import de.rki.coronawarnapp.dccticketing.core.transaction.DccTicketingVerificationMethod
 import timber.log.Timber
 
+/**
+ * @throws [DccTicketingException] if there is any element in [DccTicketingServiceIdentityDocument.verificationMethod]
+ * where [DccTicketingVerificationMethod.publicKeyJwk] is set and [DccJWK.x5c] does not contain at least one element,
+ * the operation shall abort with the specified error code, otherwise the document is valid
+ */
+fun DccTicketingServiceIdentityDocument.verifyJwks(emptyX5cErrorCode: DccTicketingErrorCode) {
```

Comment:
isn't the `emptyX5cErrorCode` always the same? or what's the reason to have it as a parameter here?

Reply:
They look similar but no , one starts with `VD` and other with `VS` 😄 , I had the same question

Reply:
:smile_cat:

Reply:
for ValidationService ist it VS and for ValidationDecorator it is VD